src/components/ChoroplethMap/ChoroplethMap.test.tsx

```
@@ -673,20 +685,40 @@ describe('ChoroplethMap animations', () => {
+  it('shows the default "Acessos" count label in the region tooltip', async () => {
+    await renderAndHoverRegion();
+
+    expect(screen.getByText(/Acessos: 200/)).toBeInTheDocument();
+  });
+
+  it('shows a custom countLabel in the region tooltip', async () => {
+    await renderAndHoverRegion({ countLabel: 'Atividades realizadas' });
+
+    expect(
+      screen.getByText(/Atividades realizadas: 200/)
+    ).toBeInTheDocument();
```

ESLint

[error] Replace `⏎······screen.getByText(/Atividades·realizadas:·200/)⏎····` with `screen.getByText(/Atividades·realizadas:·200/)`

(prettier/prettier)